generator/src/main/java/com/scottlogic/deg/generator/inputs/validation/messages/NumericConstraintValidationMessages.java

```diff
@@ -0,0 +1,28 @@
+package com.scottlogic.deg.generator.inputs.validation.messages;
+
+import java.math.BigDecimal;
+
+public class NumericConstraintValidationMessages implements StandardValidationMessages {
+
+
+    private BigDecimal validMin;
+    private BigDecimal validMax;
+    private BigDecimal invalidValue;
+
+    public NumericConstraintValidationMessages(BigDecimal validMin, BigDecimal validMax, BigDecimal invalidValue) {
+
+        this.validMin = validMin;
+        this.validMax = validMax;
+        this.invalidValue = invalidValue;
+    }
+
+    @Override
+    public String getVerboseMessage() {
+
+        return String.format(
+            "Numeric constraint with value %s is not valid. The valid range is between %s and %s.",
```

Comment:
Would this mean you could get either of the messages below:

"Numeric constraint with the value 10 is not valid. The valid range is between anything and 9"

OR

"Numeric constraint with the value 10 is not valid. The valid range is between 11 and more"

Not sure they make a huge amount of sense.

Maybe something like:

"Numeric constraint with value is not valid it must be [>= 11] [and] [<= 9]"

Reply:
you are right, thanks. changed.